rfcs/convergence/authoring-stories.md

```diff
@@ -373,26 +357,15 @@ export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button
 
 ### 8. location and naming convention
 
-1. Story files should be in the same folder as the component which they are meant to show off. This will usually be the component which is the most unique to the story.
-2. Name of every .stories.tsx file should start with name of a component which it is targeting, to improve colocation in alphabetical file ordering. If appropriate, story name should be adjusted via configuration to a well readable name, suitable for documentation, like this:
-
-```tsx
-export const ButtonPrimary = (props: ButtonProps) => <Button {...props}>Text</Button>;
-ButtonPrimary.args = {
-  primary: true,
-};
-ButtonPrimary.storyName = 'Better story name';
-```
-
-3. If an internal story is broken out into its own file, then the file should mirror the name of the story, which will include the `Internal` keyword and a `.internal.stories` suffix for easier IDE search.
+All `.stories.tsx` which are related to a single docs page files must be in the same folder, which can either be the folder with component code, or a folder called `stories`.
```

Comment:
We should standardize this and add a specific example. With the separate story files, IMO the stories and markdown docs ought to go in a subfolder to make it easier to find the actual component files.

I'd suggest either this:
```
src/
  components/
    ComponentName/
      stories/
```
or possibly:
```
src/
  stories/
    ComponentName/ (maybe?)
```

Reply:
I agree as well, it is better if we have one way of file structure. IMO, the second option is clearer:
`src/ 
  stories/`